bundle/edu.gemini.qv.plugin/src/main/scala/edu/gemini/qv/plugin/filter/core/Filter.scala

```diff
@@ -242,7 +242,7 @@ object Filter {
   // =======================  *** Set time filter ***
   case class SetTime(ctx: QvContext, min: Double = 0, max: Double = 15) extends SimpleRangeFilter {
     def label = "Set Time Hrs"
-    def getter: Obs => Double = { o => TimeUtils.asHours(SolutionProvider(ctx).remainingHours(ctx, o)) }
+    def getter: Obs => Double = { o => TimeUtils.asHours(SolutionProvider(ctx).remainingHours(ctx, o).getOrElse(0L)) }
```

Comment:
Kind of hard to tell from the code but maybe it should default to `min` since the description claims the value will be in `[min, max)` ... ?